modin/pandas/dataframe.py

```diff
@@ -2197,21 +2197,24 @@ def to_parquet(
         partition_cols=None,
         storage_options: StorageOptions = None,
         **kwargs,
-    ):  # pragma: no cover # noqa: PR01, RT01, D200
-        """
-        Write a ``DataFrame`` to the binary parquet format.
-        """
-        return self._default_to_pandas(
-            pandas.DataFrame.to_parquet,
-            path,
-            engine=engine,
-            compression=compression,
-            index=index,
-            partition_cols=partition_cols,
-            storage_options=storage_options,
-            **kwargs,
+    ):
+
+        config = {
+            "path": path,
+            "engine": engine,
+            "compression": compression,
+            "index": index,
+            "partition_cols": partition_cols,
+            "storage_options": storage_options,
+        }
+        new_query_compiler = self._query_compiler
```

Comment:
What is this assignment for?

Reply:
This is similar to to_csv where we get the query compiler from the class itself then pass it to the Factory Dispatcher. In theory, `self._query_compiler` could be passed directly to the factory dispatcher without this assignment, but I think this is a bit cleaner and is similar to other portions of the codebase.

Reply:
Okay, I see